.github/workflows/release.yml

```diff
@@ -34,30 +34,30 @@ jobs:
       - uses: taiki-e/upload-rust-binary-action@v1
         with:
           # (required)
-          bin: holochain-runner
+          bin: acorn
```

Comment:
this instruction is relevant if this repository itself is a rust crate, but in this case, it's not so it can be removed